packages/sdk-core/scripts/ajv.ts

```diff
@@ -90,39 +85,26 @@ const streamRequestSchema: JSONSchemaType<IStreamRequestFilter> = {
     },
 };
 
-function wrapValidationWithCustomError<T>(
-    validateFunction: ValidateFunction<T>
-) {
-    return (filter: T) => {
-        if (!validateFunction(filter)) {
-            throw new SFError({
-                type: "INVALID_OBJECT",
-                customMessage: "Invalid Filter Object",
-                errorObject: validateFunction.errors,
-            });
-        }
-    };
-}
+const ajv = new Ajv({
+    schemas: [
+        superTokenRequestSchema,
+        eventRequestSchema,
+        indexRequestSchema,
+        accountTokenSnapshotRequestSchema,
+        indexSubscriptionRequestSchema,
+        streamRequestSchema,
+    ],
+    formats: require("./ajvCustomFormats.js"),
+    code: {
+        lines: true, // Please format after generation.
+        optimize: true,
+        source: true,
+        esm: true,
+        formats: _`require("./ajvCustomFormats.js")`,
+    },
+});
+
+let moduleCode = standaloneCode(ajv);
```

Comment:
just out of curiosity, where did ya get the idea to do it like this

Reply:
The community member recommended it who was having the `unsafe-eval` issue with the Chrome Extension.

Reply:
gotcha